frontend/src/pages/chat/Chat.tsx

```diff
@@ -93,6 +94,10 @@ const Chat = () => {
     }, [appStateContext?.state.chatHistoryLoadingState])
 
     const getUserInfoList = async () => {
+        if (AUTH_ENABLED === "false") {
```

Comment:
The README suggests "False" - let's make this less fragile by having the backend either call .lower() or pass a bool instead of a string